src/lights/PointLightShadow.js

```diff
@@ -84,7 +38,48 @@ PointLightShadow.prototype = Object.assign( Object.create( LightShadow.prototype
 
 	}
 
-} );
+}
 
+PointLightShadow.prototype.isPointLightShadow = true;
+PointLightShadow.prototype._cubeDirections = [
+	new Vector3( 1, 0, 0 ), new Vector3( - 1, 0, 0 ), new Vector3( 0, 0, 1 ),
+	new Vector3( 0, 0, - 1 ), new Vector3( 0, 1, 0 ), new Vector3( 0, - 1, 0 )
+];
+PointLightShadow.prototype._frameExtents = new Vector2( 4, 2 );
```

Comment:
In `LightShadow`, properties like `_frameExtents` or `_viewportCount` were assigned to `this`. Why are you assigning these properties here to `prototype`? 

Notice that `_frameExtents` is not a constant value. It will be changed in the renderer. Assigning these properties to `this` should fix the broken example.

Reply:
thank you. these variables with the _* were causing errors with the way lights were compared in the unit tests. will adjust and we can go from there

Reply:
yeah, so, the way lights are compared atm, any of these `_*` properties being put into the constructor will fail the [unit tests](https://github.com/mrdoob/three.js/pull/20018/checks?check_run_id=996486782).